homeassistant/const.py

```diff
@@ -670,8 +670,19 @@ class UnitOfPower(StrEnum):
 )
 """Deprecated: please use UnitOfPower.BTU_PER_HOUR."""
 
+
 # Reactive power units
-POWER_VOLT_AMPERE_REACTIVE: Final = "var"
+class UnitOfReactivePower(StrEnum):
+    """Reactive power units."""
+
+    VOLT_AMPERE_REACTIVE = "var"
+
+
+_DEPRECATED_POWER_VOLT_AMPERE_REACTIVE: Final = DeprecatedConstantEnum(
+    UnitOfReactivePower.VOLT_AMPERE_REACTIVE,
+    "2025.1",
```

Comment:
```suggestion
    "2026.8",
```

The deprecation period for things, which cannot be fixed by the end user is one year

Reply:
Ok, I didn't know the exact time span, so I took 6 month. Besides, one year would be 2025.8.